src/components/Setup/Views/InstallPWA.tsx

```
@@ -1,182 +1,164 @@
+
     const { user } = useAuth()
     const { push } = useRouter()
 
-    // Use the prop if provided, otherwise detect locally
-    const [isUnsupportedBrowser, setIsUnsupportedBrowser] = useState(false)
+    const [installCancelled, setInstallCancelled] = useState(false)
+    const [isInstallInProgress, setIsInstallInProgress] = useState(false)
 
-    // Redirect to home if user is logged in
     useEffect(() => {
         if (!!user) push('/home')
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)